File: October, 2024/Oct_7_Leet2696/oct7.py
class Solution:
    def minLength(self, s: str) -> int:
        while s.find('AB')>-1 or s.find('CD')>-1:
            if s.find('AB')>-1:
                s = s.replace("AB", '')
            if s.find('CD')>-1:
                s = s.replace("CD", '')
        return len(s)
        

File: October, 2024/Oct_7_Leet2696/oct7.java
class Solution {
    public int minLength(String s) {
        // StringBuilder sb = new StringBuilder(s);
        while(s.contains("AB") || s.contains("CD")){
            if(s.contains("AB")){
                s=s.replace("AB", "");
            }
            if(s.contains("CD")){
                s=s.replace("CD", "");
            }
        }
        return s.length();
    }
}